notebooks/photutils/01_background_estimation/01_background_estimation.ipynb

```diff
@@ -0,0 +1,659 @@
+{
```

Comment:
Since the specifics for adding the colorbars differ for plots with one or two subplots, I'm inclined not to make it into a function - since I would need to add cases into that function to distinguish between the 1- and 2-subplot cases.

Reply:
Ah, that makes sense. I didn't read it closely enough 😀